app/code/Magento/Newsletter/Controller/Subscriber/NewAction.php

```diff
@@ -125,6 +129,17 @@ protected function validateGuestSubscription()
         }
     }
 
+    /**
+     * Check subscription is enabled in configuration
+     *
+     * @return bool
+     */
+    protected function validateIsSubscriptionEnabled()
+    {
+        return (bool)$this->_objectManager->get(ScopeConfigInterface::class)
```

Comment:
We should avoid using objectManager. Please inject and use \Magento\Newsletter\Model\Config::isActive instead.